core/src/main/resources/META-INF/upgrade/AccessControlled.hint

```diff
@@ -0,0 +1 @@
+$ac.getACL().hasPermission($a, $p) :: $ac instanceof hudson.security.AccessControlled && $a instanceof org.acegisecurity.Authentication && $p instanceof hudson.security.Permission => $ac.hasPermission($a, $p);;
```

Comment:
Do these file get autogenerated?
We have no guidelines about adding them, but it may be useful.